### DISCUSSION TITLE
pip install open-webui directory path

### DISCUSSION THREAD

**Discussion by tjdalsry** [open-webui/open-webui#3608] [Q&A]
I installed open-webui using pip install. 
I don't know where the directory path for open-webui is.
When installing with npm, there was a separate open-webui directory, but it is not visible now.

**Comment by CBS9564** [comment]
up

**Comment by kelheor** [comment]
I found this:
C:\Users\USERNAME\anaconda3\envs\ENVIRONMENT_NAME\Lib\site-packages\open_webui
